src/Microsoft.DotNet.Build.Tasks/Targets/toolruntime.targets

```diff
@@ -24,9 +24,22 @@
     <ToolHostCmd Condition="'$(COMPLUS_InstallRoot)' != ''">(set COMPLUS_InstallRoot=) &amp; $(ToolHostCmd)</ToolHostCmd>
   </PropertyGroup>
 
+  <!-- If BUILDTOOLS_TOOLS_RUNTIME is set, then we will use that as the tool and test runtime, else we will restore it from NuGet. -->
+  <Target Name="UseBuildToolsRuntimeFromEnvVariable"
```

Comment:
I would like us to have a way to make it easy for someone to bring just the native components instead of having to build out an entire tool runtime.  Otherwise folks are going to have to fight with NuGet to pull down all the managed stuff + our tools.